README.md

```diff
@@ -46,70 +130,142 @@ Key scripts include:
+#
 
-## 👥 Meet the Team
+3. **Create a new branch or use the main branch:** When modifying contracts kindly make sure the formatting is correct and all tests pass successfully.
 
-### Omar Hurtado Munguia
-**Co-founder**
+    - Create a branch name based on the type of change (e.g., `feat/name-related-issue`, `docs/name-related-issue`).
 
-Omar's journey from coffee picker to coffee tour guide in Poás de Alajuela inspired the creation of CofiBlocks. He brings firsthand knowledge of the challenges small producers face.
+    ```
+    git checkout -b branch-name
+    ```
+    - One of ideas on how to implement it for the branch name:
 
-### Karla Córdoba Brenes
-**Co-founder**
+        > `docs/update-readme` or `fix/bottom-bug`.
 
-Karla contributes her extensive experience in blockchain technology and impact-driven solutions.
+#
 
-### Ranulfo Paiva Sobrinho
-**Co-founder**
+4. **Commit:** Commit your changes.
 
-Ranulfo's background in blockchain development and collaborative economies helps drive the technical and organizational aspects of CofiBlocks.
+    1. **git add (file-name)**
+    2. **git commit -m "[type] description"**
+
+    - Example: 
+    ```
+    `git add Create_Documentation`
+
+    `git commit -m "[docs]: update documentation"`
+    ```
+
+#
+
+5. **Push fork:** Push to your fork and submit a pull request on our `main` branch. Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
+
+- Push your changes to your forked repository:
+    ```bash
+   git push origin your-branch-name
+   ```
+   > Replace `your-branch-name` with the name of your branch.
+
+- Example: 
+
+    ```bash
+    `git push origin fix/bug-fix`
+    ```
+
+#
+
+6. **Submit a Pull Request:** Submit a pull request to the `main` branch of the Semaphore Stellar SDK repository.
+
+    - <a href="https://github.com/Vagabonds-Labs/cofiblocks/pulls" target="_blank"> Summit pull request</a>
+
+# **📁 Commits**
+
+You can do a regular commit by following the next:
+
+``` [type] significant message ```
+
+- <a href="https://www.conventionalcommits.org/en/v1.0.0/" target="_blank">Learn more about conventional commits</a>
+
+### Type
+
+**The type must be one of the following:**
+
+- feat: A new feature.
+- fix: A bug fix.
+- docs: Documentation only changes.
+- style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc).
+- refactor: A code change that neither fixes a bug nor adds a feature (improvements of the code structure).
+- perf: A code change that improves the performance.
+- test: Adding missing or correcting existing tests.
+- build: Changes that affect the build system or external dependencies (example scopes: gulp, npm).
+- ci: Changes to CI configuration files and scripts (example scopes: travis, circle).
+- chore: Other changes that don't modify src or test files.
+- revert: Reverts a previous commit.
```

Comment:
Since there is already a link to conventional commits can you just do a few examples of types?